src/avatar.jsx

```diff
@@ -86,6 +83,10 @@ const Avatar = React.createClass({
     this.setState({muiTheme: newMuiTheme});
   },
 
+  getTheme() {
```

Comment:
Can we remove the usage of `getTheme` method from this PR? It's always called multiple times inside the same function. That seems inefficien.